.github/workflows/ci.yml

```
@@ -256,7 +256,7 @@ jobs:
           grep -q 'yt_dlp_ejs/yt/solver/lib\.min\.js' .wheel_contents
       - name: Install Bun requirements
         run: |
-          bun install
+          python pnpm.py install --frozen-lockfile
       - name: Bundle with Bun
         run: |
           bun --bun run bundle
```

Same here, except there is a problem now: `node` is installed on the runner and has priority in the build script. Should `pnpm.py` have a flag to force a specific runtime?
```suggestion
          python pnpm.py run bundle
```